sysutils/diskus/Portfile

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        sharkdp diskus 0.5.0 v
+
+categories          sysutils
+platforms           darwin linux
+maintainers         {@ylluminarious orbitalimpact.com:georgedp} openmaintainer
+license             {MIT Apache-2}
+
+description         A minimal, fast alternative to 'du -sb'
+
+long_description    diskus is a very simple program that computes the apparent \
+    size of the current directory. It is a parallelized version of du -sb. On \
+    the developer's 8-core laptop, it is about 10x faster than du with a cold \
+    disk cache and more than 3x faster with a warm disk cache. Currently, it \
+    only computes apparent size, not resident size.
+
+checksums           diskus-0.5.0.tar.gz \
```

Comment:
This should probably be something like `${distname}${extract.suffix}`? I didn't test though.

Reply:
@mojca Done! Thanks for the tip!